pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -57,7 +60,7 @@ let
     in attrs: concatStringsSep " " (attrValues (mapAttrs toFlag attrs));
 
   gnSystemLibraries = [
-    "flac" "libwebp" "snappy" "yasm"
+    "ffmpeg" "flac" "harfbuzz-ng" "libwebp" "libxslt" "yasm" "snappy" # "libpng" "libjpeg"
```

Comment:
They are both causing build errors. If somebody were to fix and integrate them, that would be awesome.